iosMath/render/MTMathUILabel.h

```diff
@@ -8,7 +8,13 @@
 //  This software may be modified and distributed under the terms of the
 //  MIT license. See the LICENSE file for details.
 //
+#if TARGET_OS_IPHONE
+// TARGET_OS_MAC is defined as 1 for both Mac OS and iOS,
+// so TARGET_OS_IPHONE is reliable.
 @import UIKit;
+#else
+@import AppKit;
```

Comment:
This seems to fail on the Travis build. Does something need to be setup to get the preprocessor macros to work correctly?